setup.py

```diff
@@ -230,6 +230,8 @@
     "tensorflow_gpu": ["tensorflow-gpu>=2.2.0,!=2.6.0,!=2.6.1"],
     "torch": ["torch"],
     "jax": ["jax>=0.2.8,!=0.3.2,<=0.3.25", "jaxlib>=0.1.65,<=0.3.25"],
+    "gcs": ["gcsfs"],
+    "gs": ["gcsfs"],
```

Comment:
Just a minor nit, AFAIU those extra dependencies are not required, right @lhoestq? Neither the 🤗Datasets examples nor the code actually needs `gcsfs`

```suggestion

```

Reply:
Yeah, I think you are right since  are no explicit references in `datasets` to gcsfs modules (and fsspec will just throw its own error here)